CRM/Mailing/Page/Browse.php

```diff
@@ -105,13 +105,13 @@ public function preProcess() {
     if (CRM_Core_Permission::check('access CiviMail')) {
       $archiveLinks = TRUE;
     }
-    if ($archiveLinks == TRUE) {
-      $this->assign('archiveLinks', $archiveLinks);
-    }
+    $this->assign('archiveLinks', $archiveLinks ?? FALSE);
```

Comment:
@eileenmcnaughton it's always set (see L67) so no need for the null-coalesce.

Reply:
@colemanw yeah - I think I'd probably removed L67 before the NULL co-erce TBH

Reply:
Not a blocker